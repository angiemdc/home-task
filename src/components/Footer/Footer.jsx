import React from 'react';
import { Logo } from '../Logo/Logo';

import './Footer.modules.scss';

/**
 * Renders the main Footer of the APP
 * @param props.info All react elements to put inside the Footer
 * @returns
 */

export const Footer = () => (
  <footer className='footer' data-testid='footerComponent'>
    <Logo />
  </footer>
);
