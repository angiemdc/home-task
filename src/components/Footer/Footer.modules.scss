@import '../../assets/styles/main.scss';

.footer {
  text-align: center;
  color: $red;
  background-color: $grey-dark;
  min-width: 100%;
  display: flex;
  flex-direction: column;
  @extend %base-text;
}
