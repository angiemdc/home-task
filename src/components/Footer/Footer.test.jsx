import React from 'react';
import { render, screen } from '@testing-library/react';
import { Footer } from './Footer';
import '@testing-library/jest-dom';

describe('Footer component', () => {
  test('Renders correctly', () => {
    render(<Footer />);

    const footer = screen.getByTestId('footerComponent');
    expect(footer).toBeInTheDocument();
    expect(footer).toMatchSnapshot();
  });
});
