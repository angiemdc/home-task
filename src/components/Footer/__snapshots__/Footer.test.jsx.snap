// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component Renders correctly 1`] = `
<footer
  class="footer"
  data-testid="footerComponent"
>
  <div
    class="logoImg"
  >
    <img
      alt="logo"
      src="netflixroulette.svg"
    />
  </div>
</footer>
`;
